shared/cfg/guest-os/Windows.cfg

```diff
@@ -27,6 +27,7 @@
     cdrom_cd1 = isos/windows/winutils.iso
 
     cpu_chk_cmd = echo %NUMBER_OF_PROCESSORS%
+    cpu_chk_all_cmd = echo %NUMBER_OF_PROCESSORS%
```

Comment:
how does this is different from `cpu_chk_cmd` for windows ?

Reply:
just avoid get cpu_chek_all_cmd in windows will return None

Reply:
in windows, it will never call `vm.get_cpu_count("cpu_chk_all_cmd")` as the check in L2364 will never succeed.

Reply:
Updated